### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by NexiusTailer**
Here are some improvements and each one is named in one of three commits:
1. Some clean up (spaces to tabs where appropriate)
2. Removed "vehicle seat != 128" check as it's deprecated code. The seat will never be 128 now in actual samp versions.
3. The code style of FIX_SilentTeleport was slightly edited to match the other fixes.inc style

### CODE REVIEW

File: fixes.inc [hunk 1]
```diff
@@ -8533,29 +8531,20 @@ native BAD_GetPlayerMenu(playerid) = GetPlayerMenu;
...
-							else
+							model = GetVehicleModel(GetPlayerVehicleID(playerid)) - 400;
+							if (_FIXES_NO_RANGE(model, 0, 211 + 1) || FIXES_gscMaxPassengers[model >>> 3] >>> ((model & 7) << 2) & 0xF == 15)
 							{
-								// They are a driver, but not in the driver's seat.
+								// In an invalid vehicle (one you can't drive).
 								FIXES_BlockUpdate(playerid);
 							}
 						}
-					#if FIX_HydraSniper
```
Comment (NexiusTailer) [inline fixes.inc:8553]: This part:
```
#if FIX_HydraSniper
	else
	{
		model = GetVehicleModel(GetPlayerVehicleID(playerid));
	}
#endif
```
was cut at all as I don't understand why we need to have GetVehicleModel and then just end that code without doing any actions with the model variable.

Comment (Y-Less) [reply fixes.inc:8553]: It's used here:

https://github.com/Open-GTO/sa-mp-fixes/blob/f2700884614c8304b063c940fe4ab141c3328abe/fixes.inc#L8567

File: fixes.inc [hunk 2]
```diff
@@ -1877,1013 +1877,1013 @@ in the output code.
 #define _FIXES_IS_UNSET(%0) ((2*%0-1+1)==-1)
 
 #if !defined FIXES_EnableAll
-	#define FIXES_EnableAll                      (0)
+	#define FIXES_EnableAll						(0)
```
Comment (Y-Less) [inline fixes.inc:1880]: These should be spaces.  Tabs are used for indentation, spaces are used for in-line alignment.

File: fixes.inc [hunk 3]
```diff
@@ -4846,21 +4846,21 @@ static stock const
...
+	#define CAM_MODE_SNIPER						(7)
+	#define CAM_MODE_ROCKETLAUNCHER				(8)
+	#define CAM_MODE_FIXED						(15)
+	#define CAM_MODE_1STPERSON					(16)
+	#define CAM_MODE_CAM_ON_A_STRING			(18)
+	#define CAM_MODE_BEHINDBOAT					(22)
+	#define CAM_MODE_CAMERA						(46)
+	#define CAM_MODE_ROCKETLAUNCHER_HS			(51)
+	#define CAM_MODE_AIMWEAPON					(53)
+	#define CAM_MODE_AIMWEAPON_FROMCAR			(55)
```
Comment (Y-Less) [inline fixes.inc:4862]: This pretty much demonstrates why - they are no longer aligned.

**Review by Y-Less — CHANGES_REQUESTED** [review]

File: fixes.inc (hunk 1 above)
Comment (NexiusTailer) [reply fixes.inc:8553]: But there is a '}' right after it so it sholud be the different bodies of code.

**Comment by NexiusTailer** [comment]
So it seems I've irrevocably deleted my own branch and thus there are some difficulties to add some changes after review in this PR. I can close this one and re-open it with required changes...

**Pull request closed without merging**